client/proc.go

```diff
@@ -32,6 +32,7 @@ type Cmd struct {
 	// If Scrub is set, the process element will automatically be removed from its anchor
 	// when the process exits.
 	Scrub bool
+	Name string
```

Comment:
I was thinking of this for showing the the name in the tag, but if the command is a one-off and will be auto-scrubbed, then there's no point in doing a name.  I agree, there's no need for this.